apps/base/utils.py

```diff
@@ -32,3 +33,11 @@ def __call__(self, instance, filename):
         filename = '{}{}'.format(uuid.uuid4(), extension)
         filename = os.path.join(self.path, filename)
         return filename
+
+
+def get_model_object(model_name, pk):
+    try:
+        model_object = model_name.objects.get(pk=pk)
+        return model_object
+    except model_name.DoesNotExist:
+        raise NotFound("{} does not exist".format(model_name.__name__))
```

Comment:
Can we have id also included in the log statement. Also can you please use single quotes here?